src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,44 @@
 import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({id, sender, body, timeStamp,liked, onLikeToggle}) => {
+
+  const handleHeartClick = () => {
+    onLikeToggle(id);
+  };
+
+  let isLocalUser = false
+
+  if (sender === 'Vladimir') {
+    isLocalUser = true;
+  } 
```

Comment:
Would prefer to have this logic put in a method and the method get called somewhere. 

```suggestion
  const setLocalOrRemote () => {
    return sender == 'Vladimir' ? 'local' : 'remote';
  };
```

Then on line 18, you can call the function:
```js
    <div className={`chat-entry ${setLocalOrRemote()}`}>
```